client/package.json

```diff
@@ -4,9 +4,12 @@
   "private": true,
   "proxy": "http://localhost:5000",
   "dependencies": {
+    "@material-ui/core": "^4.9.9",
+    "@material-ui/icons": "^4.9.1",
```

Comment:
if you made pull to the master , you will found that they are already installed ^_^